### DISCUSSION TITLE
Beautiful theme, but line interspacing too large

### DISCUSSION THREAD

**Discussion by SanderBouwhuis** [witalihirsch/qBitTorrent-fluent-theme#27] [Ideas]
I absolutely LOVE your theme for Qbittorrent, but the vertical (line) interspacing is so much, that I can only see about half as many torrents as I used to.

Could you drastically reduce the distance between lines so that we can see more?

**Comment by witalihirsch** [comment]
slightly reduced the size of the torrents, but I can’t anymore, it'll violate the concept of the theme

**Comment by SanderBouwhuis** [comment]
Thanks, I'll give it a go.
